cmd/deploy/command_configuration.go

```diff
@@ -35,6 +36,14 @@ const (
 )
 
 func setDeployOptionsValuesFromManifest(ctx context.Context, deployOptions *Options, cwd string, c kubernetes.Interface) error {
+	if deployOptions.RunInRemote {
```

Comment:
Can we cover these new lines in the unit test?

Reply:
discussed that for now we do not introduce test on it, testing that function is out of the scope of the PR.